pkg/gui.py

```
@@ -143,13 +160,16 @@ class GUI(App):
-        super(GUI, self).__init__(parent, cmd_queue, title)
+    
+    MAX_HISTORY_ITEMS = 5
+
+    def __init__(self, parent: Tk, gui_queue: Queue, cmd_queue: Queue, title: str) -> None:
+        super().__init__(parent, cmd_queue, title)
         # Base Settings
         parent.minsize(300, 90)
         parent.maxsize(1000, 800)
         self.alive_time = 6000
-        self.since_last_popup = time.now()
+        self.since_last_popup = datetime.now()
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)